docs/index.rst

```diff
@@ -82,7 +82,6 @@ Discover Neptune
 - |Example Project|: See how example project looks in Neptune
 - |YouTube channel|: Provides hands-on videos that showcase key Neptune features.
 - |Neptune blog|: Provides in-depth articles about best practices in machine learning experimentation (among other things)
-- |Neptune community|: Meet other Neptune users and developers and start a discussion.
```

Comment:
also drop link definition at the bottom of the file.

Reply:
**done**